### DISCUSSION TITLE
Bulk Remove Tags

### DISCUSSION THREAD

**Discussion by twilson-ay** [thoughtspot/cs_tools#190] [Q&A] [answered]
Using the Archiver tool I can bulk add tags to content. Is there a way to bulk remove tags as well?

**Comment by boonhapus** [comment, marked as answer]
@twilson-ay You can delete the tag with the **[V2.0 API Playground](https://developers.thoughtspot.com/docs/restV2-playground?apiResourceId=http%2Fapi-endpoints%2Ftags%2Fdelete-tag)** or you can run `cs_tools tools archiver revert --tag {name} --delete-tag` as well.

**Reply by twilson-ay** [reply]
Thanks for the quick reply! This worked perfectly.
